src_frontend/store_backend.ts

```diff
@@ -13,7 +13,8 @@ function transformValue(x: any): any {
 }
 
 export class StoreBackend implements StoreInterface {
-  url = "http://localhost:5000";
+  // TODO: this needs to be dynamic
+  url = "http://127.0.0.1:5000"
```

Comment:
This needs to be dynamic in some way. Could we guess the address of the local device on startup? or we're obliged to get it from some sort of config?